### ISSUE TITLE
The property 'line' is set to 2 which is out of range

### ISSUE THREAD

**Issue by bmos** [kbrashears5/github-action-file-sync#17]
I get this error when I use 2.0.11 (which is marked as the latest version although the tags on this repo are confusing as mentioned in issue #13 ):
> Error: Unable to process command ['::add-matcher::/github/home/file-sync-error-matcher.json'](https://github.com/kbrashears5/github-action-file-sync/blob/cfc836e9702ccd4a70afc7a503aa19057d352a24/entrypoint.sh#L10) successfully.
> Error: The property 'line' is set to 2 which is out of range

my file:
```
name: File Sync
 
 on:
   push:
     paths:
       - 'fg-luacheck-action.yml'
       - 'sync-luacheck.yml'
   schedule:
     - cron: 0 0 * * *
   workflow_dispatch:
 
 jobs:
   file_sync:
     runs-on: ubuntu-latest
     steps:
       - name: Fetching Local Repository
         uses: actions/checkout@master
       - name: Syncing File
         uses: kbrashears5/github-action-file-sync@2.0.11
         with:
           REPOSITORIES: |
             FG-Unofficial-Developers-Guild/FG-Aura-Effect@main
           FILES: |
             fg-luacheck-action.yml=.github/luacheck.yml
           TOKEN: ${{ secrets.ACTIONS }}
```

**Comment by ghost** [comment]
#### AUTOMATICALLY GENERATED by kbrashears5/github-action-issue-notifier

Mentioning @kbrashears5 to notify them. This will be looked at soon!

**Comment by bmos** [comment]
It works great, btw.
The error is not keeping it from working.
However, it does send me notifications that it has failed.

**Comment by bmos** [comment]
```
/usr/bin/docker run --name a95e0ac338738ee3485f983cd20f8d6266ee_7f6534 --label 29a95e --workdir /github/workspace --rm -e INPUT_REPOSITORIES -e INPUT_FILES -e INPUT_TOKEN -e INPUT_PULL_REQUEST_BRANCH_NAME -e INPUT_GIT_EMAIL -e INPUT_GIT_USERNAME -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/fg-workflows/fg-workflows":"/github/workspace" 29a95e:0ac338738ee3485f983cd20f8d6266ee
Error: Unable to process command '::add-matcher::/github/home/file-sync-error-matcher.json' successfully.
Error: The property 'line' is set to 2 which is out of range
Repository: [bmos/fg-workflows]
Inputs
---------------------------------------------
Repositories    : FG-Unofficial-Developers-Guild/FG-4E-TEMPO-and-TEMPA@main
Files           : luacheck.yml=.github/workflows/luacheck.yml
Pull request    : 
Git email       : 41898282+github-actions[bot]@users.noreply.github.com
Git username    : github-actions[bot]
Temp Path       : /ghafs/
```

**Comment by bmos** [comment]
Looking at error-matcher.json, it seems that version [2.0.11](https://github.com/kbrashears5/github-action-file-sync/blob/2.0.11/error-matcher.json) sets the line property to 2 whereas version [2.0.10](https://github.com/kbrashears5/github-action-file-sync/blob/2.0.10/error-matcher.json) does not. This [has been fixed](https://github.com/kbrashears5/github-action-file-sync/blob/master/error-matcher.json) on May 24, 2021, but the fix was never tagged so the only workaround is to target the master branch.

**Comment by ghost** [comment]
This issue has not received a response in a timely manner. If you still think there is a problem, please leave a comment to avoid the issue from automatically closing.

**Comment by bmos** [comment]
This is still a problem.

**Comment by ghost** [comment]
This issue has not received a response in a timely manner. If you still think there is a problem, please leave a comment to avoid the issue from automatically closing.

**Comment by bmos** [comment]
This is still a problem.

**Comment by ghost** [comment]
This issue has not received a response in a timely manner. If you still think there is a problem, please leave a comment to avoid the issue from automatically closing.

**Comment by bmos** [comment]
This is still a problem.

**Comment by bmos** [comment]
This is still a problem.

**Comment by ghost** [comment]
This issue has not received a response in a timely manner. If you still think there is a problem, please leave a comment to avoid the issue from automatically closing.

**Comment by bmos** [comment]
This is still a problem.

**Comment by bodgit** [comment]
Bumping into this too. @kbrashears5 any chance you can push a new release, ideally sorting the tags out too given there's a v2.2.0 that seems to be older than 2.0.11.

**Comment by bodgit** [comment]
@bmos for now I've ended up using 2.0.10 (which is only missing the commit included in 2.0.11 that introduced this error) and I'm using the following dependabot configuration:
```yaml
version: 2

updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
    ignore:
      - dependency-name: "kbrashears5/github-action-file-sync"
        versions:
          - "2.0.11"
          - "2.1.0"
          - "2.2.0"
```
so it won't try and update to a broken (or older!) version.

**Comment by kbrashears5** [comment]
sorry for the delay, I've been off github for a while. please use v3.0.0, my auto releases got out of order
